inspirehep/modules/records/mappings/records/hep.json

```diff
@@ -20,7 +20,20 @@
                     "type": "pattern_replace",
                     "pattern": "[ \t]{2,}",
                     "replacement": " "
-                }
+                },
+                "tc_synonyms" : {
+                    "type" : "synonym",
+                    "synonyms" : [
+                        "b => book",
+                        "c => conferencepaper",
+                        "experimental note => note",
+                        "p => published",
+                        "t => thesis",
+                        "i => introductory",
+                        "l => lectures",
+                        "r => review"
+                      ]
+                    }
```

Comment:
:children_crossing: **warning** :children_crossing:

Since we are touching the mappings, we _might_ have to reindex when this is merged.